docs/source/development.rst

```diff
@@ -28,15 +28,19 @@ We do not have a concrete plan about versioning strategy after v1.0.0.
 Compatibiity policy
 ===================
 
-As an immediate consequence of the semantic versioning, we may break compatibility of public APIs including addition, deletion, change in semantics of them anytime in the major version zero.
-Since APIs of Chainer Chemistry are still immature and unstable, introduction of new features can sometime involve compatibility break.
+As an immediate consequence of the semantic versioning, we may break compatibility of public APIs including addition, deletion, and changes in their semantics anytime in the major version zero.
+Actally, since APIs of Chainer Chemistry are still immature and unstable, we expect introduction of new features can sometime involve compatibility break.
```

Comment:
typo: Actually

But I think "Actually" is not necessary (original sentense is ok I think)

Reply:
Reverted